README.md

```diff
@@ -59,6 +59,7 @@ $ php artisan vendor:publish --provider="AvtoDev\FakerProviders\Frameworks\Larav
 - Объект IDEntity (необходима установка дополнительного пакета [avto-dev/identity-laravel][identity]).
 - Ссылку на аватар пользователя
 - ИНН и КПП организации
+- Кадастровый номер объекта недвижимости
```

Comment:
Minimal required version for `avto-dev/identity-laravel` in `composer.json` must be updated

Reply:
`changed`